src/test/java/core/basesyntax/serviceimpl/ProcessServiceImplTest.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.serviceimpl;
+
+import static core.basesyntax.db.Storage.storage;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.BalanceOperation;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.PurchaseOperation;
+import core.basesyntax.operation.ReturnOperation;
+import core.basesyntax.operation.SupplyOperation;
+import core.basesyntax.strategy.StrategyImpl;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class ProcessServiceImplTest {
+    private static final List<FruitTransaction> TRANSACTIONS_TO_PROCESS = List.of(
+            new FruitTransaction(FruitTransaction.Operation.BALANCE,
+                    "banana",
+                    20),
```

Comment:
if it doesn't fit in one line, let's import enum constant, because it looks messy